server/backend/test/steps/send-notification.steps.ts

```diff
@@ -155,23 +166,32 @@ defineFeature(feature, (test) => {
       cameraA = camera;
       cameraA.group = new Group('group');
       cameraA.group.user = new User();
+      cameraA.group.user.expoToken = 'ExponentPushToken[000000000000]';
       await cameraRepository.persistAndFlush(cameraA);
       token = getCameraAuthToken(cameraA, keyPair.privateKey);
     });
-    and('Camera A has 1 notification', async () => {
-      cameraA.notifications.add(new Notification());
-      await cameraRepository.flush();
+    and('Camera A has 0 notifications', async () => {
+      const res = await request(app.getHttpServer())
+        .get(`/cameras/${cameraA.id}/notifications`)
+        .set('Authorization', token);
+      expect(res.status).toBe(200);
+      expect(res.body).toHaveLength(0);
     });
     when('I try to send a notification on behalf of camera A', async () => {
       sendRes = await request(app.getHttpServer())
         .put(`/cameras/${cameraA.id}/notifications`)
-        .set('Authorization', token);
+        .set('Authorization', token)
+        .send(new Notification());
```

Comment:
A `Notification`object isn't what the backend is expecting from clients, so we shouldn't send one here. We're not handling frame data at the moment, but for reference, [this](https://github.com/tucker-moore/human-detector/blob/main/camera/detector/src/networking/net_requests.py#L17-L19) is what the client is sending.
```suggestion
        .send();
```